bot/utils/helpers.py

```diff
@@ -6,3 +6,26 @@ def suppress_links(message: str) -> str:
     for link in set(re.findall(r"https?://[^\s]+", message, re.IGNORECASE)):
         message = message.replace(link, f"<{link}>")
     return message
+
+def neutralise_string(txt: str) -> str:
+    """Attempts to neutralise all punctuation and cases and returns a string of lowercase words."""
+    # take out punctuation
```

Comment:
This currently doesn't take a `None` input into account.

If you have a look at the `randomcase` command, the bot will return a usage embed when a text isn't provided, so you should do the same here by exiting early and let the `clean_text_or_reply`  raise a `BadArgument` exception